packages/contracts/tasks/verify-blockscout.js

```diff
@@ -13,12 +13,12 @@ const BLOCKSCOUT_CHAIN_IDS = [77, 100];
 
 const API_URLS = {
   77: "https://blockscout.com/poa/sokol/api",
-  100: "https://blockscout.com/xdai/mainnet/api",
+  100: "https://gnosis.blockscout.com/api",
```

Comment:
I'm guessing our hardhat version is going to be too old to verify with this url. But maybe that's ok and we can just verify manually. Good incentive to also update the hardhat version, or even Foundry, but those are big and bigger lifts respectively.